modules/rosa-create-objects.adoc

```diff
@@ -6,9 +6,55 @@
 [id="rosa-create-objects_{context}"]
 = Create objects
 
-
 This section describes the `create` commands for clusters and resources.
 
+[id="rosa-create-account-roles_{context}"]
+== create account-roles
+
+Create the needed account-wide role and policy resources for your cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create account-roles [flags]
+----
+
+.Flags
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--debug
+|Enable debug mode.
+
+|-i, --interactive
+|Enable interactive mode.
+
+|-m, --mode string
+|How to perform the operation. Valid options are:
+    auto: Resource changes will be automatic applied using the current AWS account
+    manual: Commands necessary to modify AWS resources will be output to be run manually
+
+|--path string
+|The ARN path for the account-wide roles and policies, including the Operator policies.
+
+|--permissions-boundary string
+|The ARN of the policy that is used to set the permissions boundary for the account roles.
+
+|--prefix string
+|User-defined prefix for all generated AWS resources (default "ManagedOpenShift")
```

Comment:
I'd recommend avoiding parentheses and saying something like "The default is `ManagedOpenShift`." But I see it's this way in other entries, so it's fine to ignore this recommendation for consistency sake!

Reply:
Thanks for highlighting this. Resolved throughout the module.